flask_redis_sentinel.py

```diff
@@ -49,6 +51,8 @@ def _connect(self):
                 sentinel_class=self.sentinel_class, sentinel_options=self.sentinel_options,
                 client_class=self.client_class, client_options=self.client_options)
             self.local.connection = conn
+            self._connections.put(conn[0])
```

Comment:
This is the only place that `None` can get into `self._connections`, isn't it? In that case I'd move the check for `None` here instead of having it in `disconnect()`